imports/collections/schemas/core.js

```diff
@@ -0,0 +1,58 @@
+import SimpleSchema from "simpl-schema";
+
+export const Money = new SimpleSchema({
+  amount: {
+    type: Number,
+    min: 0
+  },
+  currencyCode: String,
+  displayAmount: {
+    type: String,
+    optional: true
+  }
```

Comment:
I did not add `displayAmount`, it was already in the existing `Money` type, I just moved the `Money` schema into the `core` file so it was easily shared, as it was in either the `Cart` or `Order` before. I can update it if we think it's worth it, but I don't know if it will break one of the other places that use the `Money` schema.

Reply:
When I look in `develop` branch I see `Money` schema in two places and neither has `displayAmount` field.

Reply:
Hmm... Ok, trying to think back on it, I definitely moved it, maybe I did add it as well, I thought I had kept it exactly the same. This was one of the initial things I changed months ago, guess I forgot exactly what I did.